2-ui/1-document/10-size-and-scroll-window/article.md

```diff
@@ -2,11 +2,11 @@
 
 How do we find the width and height of the browser window? How do we get the full width and height of the document, including the scrolled out part? How do we scroll the page using JavaScript?
 
-For most such requests, we can use the root document element `document.documentElement`, that corresponds to the `<html>` tag. But there are additional methods and peculiarities important enough to consider.
+For these, we can use the root document element `document.documentElement`, which corresponds to the `<html>` tag. But there are additional methods and peculiarities to consider.
```

Comment:
I would leave this as is, or maybe replace "most such requests," with "for this type of information" (probably a little better w/out comma). And I prefer "that" over "which".

Reply:
Breaking the "that" clause out into a separate sentence doesn't seem as good to me as way it is in the original. Still favor my original suggestion.

Reply:
```suggestion
For this type of information, we can use the root document element `document.documentElement`, that corresponds to the `<html>` tag. But there are additional methods and peculiarities to consider.
```